nixos/modules/system/boot/systemd.nix

```diff
@@ -858,7 +858,12 @@ in
       "sysctl.d/50-coredump.conf".source = "${systemd}/example/sysctl.d/50-coredump.conf";
       "sysctl.d/50-default.conf".source = "${systemd}/example/sysctl.d/50-default.conf";
 
+      "tmpfiles.d/journal-nocow.conf".source = "${systemd}/example/tmpfiles.d/journal-nocow.conf";
+      "tmpfiles.d/static-nodes-permissions.conf".source = "${systemd}/example/tmpfiles.d/static-nodes-permissions.conf";
       "tmpfiles.d/systemd.conf".source = "${systemd}/example/tmpfiles.d/systemd.conf";
+      "tmpfiles.d/systemd-nspawn.conf".source = "${systemd}/example/tmpfiles.d/system-nspawn.conf";
+      "tmpfiles.d/systemd-tmp.conf".source = "${systemd}/example/tmpfiles.d/system-tmp.conf";
+      "tmpfiles.d/var.conf".source = "${systemd}/example/tmpfiles.d/var.conf";
```

Comment:
We probably should add some logic extracting tmpfiles.d from common locations into the systemd module, like we already do for udev files for everything added to `systemd.packages`.

This would be useful for https://github.com/NixOS/nixpkgs/pull/66856#discussion_r315065596, too.

Reply:
That is a great idea @flokli . Lets open a new issue for that?

Reply:
Seems there already is an issue for that: https://github.com/NixOS/nixpkgs/issues/40594